src/main/resources/application.yaml

```diff
@@ -1,6 +1,10 @@
 server:
   port: 4750
 
+spring:
+  main:
+    allow-circular-references: true
```

Comment:
Out of interest, why is this needed?

Reply:
It was, for some reason. Removed and no longer needed